inst/templates/app_structure/app/main.R

```diff
@@ -1,13 +1,17 @@
 box::use(
-  shiny[bootstrapPage, moduleServer, NS, renderText, tags, textOutput],
+  shiny[bootstrapPage, div, moduleServer, NS, renderText, tags, textOutput],
 )
 
 #' @export
 ui <- function(id) {
   ns <- NS(id)
   bootstrapPage(
-    tags$h3(
-      textOutput(ns("message"))
+    div(
+      style = "display: flex; justify-content: center; align-items: center; height: 100vh;",
+      tags$h1(
+        textOutput(ns("message")),
+        tags$a("Check out Rhino docs!", href = "https://appsilon.github.io/rhino/")
```

Comment:
To keep the example minimal, perhaps drop the "Hello!" and render the link? Either with `uiOutput()` (render the whole link) or `textOutput(inline = TRUE)` (render just the link text). The `test-main.R` would need an update.

Reply:
I have chosen `uiOutput` and dropped "Hello".